### PR TITLE
admin: some improvements 

### PR DESCRIPTION
**Pull request by iDannz1**
When merged, this PR will:

- Merge the PositionX, PositionY, PositionZ labels into one.

- Move and re-scale "unread messages" and "screenshots" buttons from the "Player" tab.

- Add new functionality: Hide Sensitive Data.
    - Useful for those who stream, share screens, or just want to share a simple screenshot. Works like "Streamer mode" in some apps, but in this case, it will never allow the admin panel to display the following data:
        - Players Tab:
            - Player serial
            - Player IP
            - Player account name
            - Player in-game localization
        - Server Tab:
            - Server password
                 
- Load and save the following data in client-side XML:
     - Anonymous Admin status
     - Hide Color Codes status
     - Hide Sensitive Data status
     
_If it's approved, ill do the same for admin2._

Before:
image

After:
image

### CODE REVIEW

File: [admin]/admin/client/gui/admin_main.lua [hunk 1]
```diff
@@ -35,11 +35,12 @@ function aAdminMenu ()
...
 		aTab1.Tab			= guiCreateTab ( "Players", aTabPanel, "players" )
-		aTab1.Messages		= guiCreateButton ( 0.75, 0.02, 0.23, 0.04, "0/0 unread messages", true, aTab1.Tab )
-		aTab1.ScreenShots		= guiCreateButton ( 0.75, 0.065, 0.23, 0.04, "screenshots", true, aTab1.Tab )
+		aTab1.Messages		= guiCreateButton ( 0.71, 0.02, 0.27, 0.04, "0/0 unread messages", true, aTab1.Tab )
+		aTab1.ScreenShots		= guiCreateButton ( 0.71, 0.065, 0.27, 0.04, "screenshots", true, aTab1.Tab )
 		aTab1.PlayerListSearch 	= guiCreateEdit ( 0.03, 0.05, 0.16, 0.04, "", true, aTab1.Tab )
 						  guiCreateStaticImage ( 0.19, 0.05, 0.035, 0.04, "client\\images\\search.png", true, aTab1.Tab )
-		aTab1.HideColorCodes= guiCreateCheckBox ( 0.037, 0.94, 0.20, 0.04, "Hide color codes", true, true, aTab1.Tab )
+		aTab1.HideColorCodes= guiCreateCheckBox ( 0.037, 0.94, 0.20, 0.04, "Hide color codes", false, true, aTab1.Tab )
+		aTab1.HideSensitiveData= guiCreateCheckBox ( 0.25, 0.94, 0.23, 0.04, "Hide sensitive datas", false, true, aTab1.Tab )
```
Comment (Dezash) [inline [admin]/admin/client/gui/admin_main.lua:43]: "data" is already plural
```suggestion
		aTab1.HideSensitiveData= guiCreateCheckBox ( 0.25, 0.94, 0.23, 0.04, "Hide sensitive data", false, true, aTab1.Tab )

```

Comment (iDannz-Breno) [reply [admin]/admin/client/gui/admin_main.lua]: ty

**Review by Dezash — COMMENTED** [review]
aAdminReloadInfos() doesn't seem to reload Server Password
Also, some ideas for sensitive data:
Disable Bans and Admin Chat tabs
Ignore "Output admin messages to chat box" setting

**Comment by Dutchman101** [comment]
Good points @Dezash

I got another one, if you /report a player and select them from the list of players to report, admins will have suspect player information attached in the report under "View info"

So @iDannz-Breno you'd need to either disable opening reports while in "Hide sensitive data" mode, or mask player serial, IP address and account name in those reports

**Review by iDannz-Breno — COMMENTED** [review]
I am not sure if this is the best way to do it, let me know if any of you have a better idea.

**Comment by iDannz-Breno** [comment]
Please let me know if you find anything else to change/add.

**Review by Dezash — COMMENTED** [review]
The reported player info is broken, I've posted a suggestion to fix it
Also, aTab1.Name width should be increased because the stripped nickname part rarely fits:
image

**Review by Dezash — APPROVED** [review]

**Comment by jlillis** [comment]
@iDannz-Breno did you want this merged as is or did you want to make the same (or similar) changes to admin2 as mentioned? These changes look good and are approved.

**Comment by iDannz-Breno** [comment]
@jlillis, I think it is better to merge it as it is, maybe next week I can submit a new PR.

**Comment by iDannz1** [comment]
Changes added to admin2 in #325.

**Review by jlillis — APPROVED** [review]
Going to approve & merge as-is for `admin` but I have some additional comments for the `admin2` implementation of this (#325).

**Pull request merged**